modules/ldsg.js

```diff
@@ -5,6 +5,40 @@ const Augur = require("augurbot-ts"),
   /** @type {string[]} */
   banned = require("../data/banned.json").features.suggestions;
 
+const hasLink = /http(s)?:\/\/(\w+(-\w+)*\.)+\w+/;
+const affiliateLinks = {
+// amazon: { //Functionality can be renabled if amazon will let us get a affiliate
+  //  site: "Amazon",
+  //  affiliate: "Amazon Affiliate",
+  //  test: /amazon\.(com|co\.uk)\/(\w+(-\w+)*\/)?(gp\/product|dp)\/(\w+)/i,
+  //  tag: /tag=ldsgamers-20/,
+  //  link: (match) => `https://www.${match[0]}?tag=ldsgamers-20`
+  //  },
+  cdkeys: {
+    site: "CDKeys.com",
+    affiliate: "CDKeys Affiliate",
+    test: /cdkeys\.com(\/\w+(-\w+)*)*/i,
+    tag: /mw_aref=LDSGamers/i,
+    link: match => `https://www.${match[0]}?mw_aref=LDSGamers`
+  },
+// humblebundle: {
+  // site: "Humble Bundle",
+  //  affiliate: "Humble Bundle Partner",
+  //  test: /humblebundle\.com(\/\w+(-\w+)*)*/i,
+  //  tag: /partner=ldsgamers/i,
+  //  link: (match) => `https://www.${match[0]}?partner=ldsgamers`
+// },
+};
+function processLinks(msg) {
```

Comment:
When we create functions, we need to use JSDoc (the /** notation) to specify the types of the parameters. This way we can see potential errors in advance, such as calling non-existent properties or functions.
```suggestion

/** @param {Discord.Message} msg */
function processLinks(msg) {
```

Reply:
So for that just say that it needs {discord.message}? Is that what that suggestion means?